crates/bevy_ecs/src/observer/runner.rs

```diff
@@ -84,7 +84,7 @@ impl Component for ObserverState {
 }
 
 /// Type for function that is run when an observer is triggered.
-/// Typically refers to the default runner that runs the system stored in the associated [`ObserverSystemComponent`],
+/// Typically refers to the default runner that runs the system stored in the associated [`Observer`] component,
```

Comment:
I didn't find any `ObserverSystemComponent`, I guess this was renamed at some point (to `Observer`? I'm not sure)

Reply:
Yes, this should just be `Observer`